### ISSUE TITLE
Implement identicon image

### ISSUE THREAD

**Issue by biocom** [gnosis/safe#41]
Created two proposals with slightly different shade accentuation:

1. https://invis.io/P7PP56T9JSZ
2. https://invis.io/8RPP5707XM9

I'd opt for going with option 1. Please review.

**Comment by tschubotz** [comment]
I prefer option 1 as well.

Which screen would be opened when tapping the identicon? The same like before or will we adapt to the iOS version a bit?

**Comment by biocom** [comment]
@tschubotz I'd envision it to be the RECEIVE screen. Is that the case on iOS?

**Comment by tschubotz** [comment]
The iOS uses the following screen, currently: https://zpl.io/a3lLkyy

**Comment by biocom** [comment]
Should we create a ticket to review/update the Android receive screen. Once that is done, @posthnikova could pick up the ticket to update iOS based on the Android view?

**Comment by tschubotz** [comment]
Yes, good idea. Could you create the ticket(s)?
From the top of my head the following 2 things are imo solved better on iOS:
* Format of the address (Highlighting of first and last 4 chars)
* The info text on top with the purpose of the address etc. ("Receive Ether and tokens ..."(
* Bigger identicon.

**Comment by fmrsabino** [comment]
I also prefer version 2. Android has been using shadows more and more.

Visually, version 2 seems clickable (compared to version 1)

**Comment by tschubotz** [comment]
I'm not too familiar with all the details of the Android guidelines. If shadows are becoming a thing, then let's use version 2.

**Comment by biocom** [comment]
As a counter point, I think arguments for going without the conventional shadow are:
- It's used already in the tabs and will potentially be used for other (floating) actions buttons in the future.
- Technically speaking the Identicon as an element, functions as a button. But it also primarily acts as an identifier, almost avatar like. So I feel using option 1 makes a good trade off between both.

At the end of the day the mentioned guidelines are exactly just guidelines. Acknowledging so that they definitely have it's apparent use case. I guess in this particular case it's not just a tab bar or (floating) action button of sorts and in my opnion *not* using the conventional shadow allows for the app to feel less 'stocky' Android.

Otherwise while favoring option 1, I don't mind if we end up with option 2, assuming we'd consider the above.

**Comment by fmrsabino** [comment]
I'd say that we can go with 1 👍

**Comment by biocom** [comment]
@tschubotz Can we close this issue and upload the Android screen for production (Zeplin)?

**Comment by tschubotz** [comment]
Yes! I wouldn't close the ticket tough, but move it to the done column on the board because we are currently thinking about how the handover and spec writing should look like and we'll probably continue using this issue (but remove the design labels)

@rmeissner this one is another issue we could write specs for.

**Comment by tschubotz** [comment]
@biocom Is the Android screen already in Zeplin? 
I think we can close the ticket as soon as it's there :)

**Comment by biocom** [comment]
@tschubotz They are available now.

Android version can be found here:
https://zpl.io/VQKk1QA

iOS:
https://zpl.io/VKKe6Aq
